### ISSUE TITLE
E-Mail templates with html/text types don't work correctly

### ISSUE THREAD

**Issue by MoritzLost** [amteich/kirby-twig#36]
I'm trying to create e-mail templates in plain text and HTML using Twig. I have created the following two templates:

- `templates/emails/contact.html.twig`
- `templates/emails/contact.text.twig`

And I'm sending an email like this:

```php
$kirby->email([
    'from' => 'info@example.com',
    'replyTo' => 'info@example.com',
    'to' => 'recipient@example.com',
    'template' => 'contact',
    'data' => [
        'foo' => 'bar',
    ],
]);
```

The e-mail is sent correctly, but the body only includes this error:

> Unable to find template "emails/contact.twig" (looked into: /path/to/project/templates, /path/to/project/templates/emails).

Looks like the `Template` class in the plugin is not handling the `$type` suffix correctly, but I'm not sure where the error comes from.

**Comment by seehat** [comment]
This is fixed in version 4.2.3.

**Comment by MoritzLost** [comment]
@seehat  Great, thanks!
